specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2019-10-01/analyticsItems_API.json

```diff
@@ -0,0 +1,380 @@
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ItemScopePathParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/OverrideItemParameter"
+          },
+          {
+            "name": "itemProperties",
+            "description": "Properties that need to be specified to create a new item and add it to an Application Insights component.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ApplicationInsightsComponentAnalyticsItem"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The new or updated item associated with the Application Insights component.",
+            "schema": {
+              "$ref": "#/definitions/ApplicationInsightsComponentAnalyticsItem"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "AnalyticsItemPut": {
+            "$ref": "./examples/AnalyticsItemPut.json"
+          }
+        }
+      },
+      "delete": {
+        "description": "Deletes a specific Analytics Items defined within an Application Insights component.",
+        "operationId": "AnalyticsItems_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ItemScopePathParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ItemIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ItemNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The item has been successfully removed from the Application Insights component"
+          }
+        },
+        "x-ms-examples": {
+          "AnalyticsItemDelete": {
+            "$ref": "./examples/AnalyticsItemDelete.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ApplicationInsightsComponentAnalyticsItem": {
+      "description": "Properties that define an Analytics item that is associated to an Application Insights component.",
+      "type": "object",
+      "properties": {
+        "Id": {
```

Comment:
It's best to reference these common types (using an allOf) from a common spec, such as: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/common-types/resource-management/v1/types.json